code/modules/mob/language.dm

```diff
@@ -238,6 +238,13 @@
 	flags = RESTRICTED|HIVEMIND
 	drone_only = 1
 
+/datum/language/zombie
+	name = "Zombie"
+	desc = "If you select this from the language screen, expect a ban."
```

Comment:
While you are at this, can you make the description a bit more friendly and thematic? We shouldn't be banning synthetics or whoever has universal language for selecting this.

Reply:
I didn't even pause to read it, but you are right. This description is very dumb.